src/u_init.c

```diff
@@ -1107,6 +1110,7 @@ u_init()
         knows_object(DWARVISH_RING_MAIL);
         knows_object(DWARVISH_CLOAK);
         knows_object(DWARVISH_ROUNDSHIELD);
+	knows_object(POT_BOOZE);
```

Comment:
Knows_object(POT_BOOZE) is actually unnecessary. As long as dwarves start with a potion of booze, then potions of booze will be type-identified by ini_inv().